api/src/main/java/com/talentpath/amazin/api/qAndA/QAndAsHardcodedService.java

```diff
@@ -0,0 +1,23 @@
+package com.talentpath.amazin.api.qAndA;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class QAndAsHardcodedService {
+
+    private static List<QAndA> qAndAs = new ArrayList<>();
+    private static long idCounter = 0;
+
+    static {
+        qAndAs.add(new QAndA(++idCounter, "5L"));
+        qAndAs.add(new QAndA(++idCounter, "10L"));
+        qAndAs.add(new QAndA(++idCounter, "15L"));
```

Comment:
I would update these to reflect the questions, answers, and upvotes as noted in my comment above